values/loki/loki.gotmpl

```diff
@@ -48,6 +48,7 @@ loki:
       max_transfer_retries: 0
       wal:
         dir: /var/loki/wal
+        replay_memory_ceiling: 512MB
```

Comment:
Maybe should be configurable, as this could bottleneck large setups and mean longer recovery time

Reply:
The Loki documentation says about replay_memory_ceiling should be around 75% of available memory, so I adjusted this values to default memory limits.
I would like to improve it further to change this value based on memory limits. However I do not have a good idea for the implementation as gotemplating is not the best to parse string and perform math. 
Would the rawValues be sufficient for time being ?

Reply:
Rawvalues is fine I guess, I think adding to the schema would just be unnecessary bloat (even if from my perspective it would be useful)